libs/website/feature/contactpage/contactFormValidation.ts

```diff
@@ -0,0 +1,48 @@
+export interface FormData {
+  name: string
+  email: string
+  subject: string
+  message: string
+}
+
+export interface FormErrors {
+  name?: string
+  email?: string
+  subject?: string
+  message?: string
+}
+
+export function validateForm(values: FormData): FormErrors {
+  const errors: FormErrors = {}
+
+  /* Name validation */
+  if (!values.name.trim()) {
+    errors.name = 'Name is required'
+  }
+  else if (values.name.length < 2) {
+    errors.name = 'Name must be at least 2 characters'
+  }
+  /* Email validation */
+  if (!values.email) {
+    errors.email = 'Email is required'
+  }
+  else if (!/^[\w.%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i.test(values.email)) {
```

Comment:
nice job on the regex, but we also need a solution that can check if the email itself is real.